manager/notifier.go

```diff
@@ -164,6 +165,38 @@ func (n *notifier) sendPagerDutyNotification(serviceKey string, a *Alert) error
 	return nil
 }
 
+func (n *notifier) sendHipChatNotification(authToken string, roomId int32, color string, notify bool, a *Alert) error {
+	// https://www.hipchat.com/docs/apiv2/method/send_room_notification
+	incidentKey := a.Fingerprint()
+	buf, err := json.Marshal(map[string]interface{}{
+		"color":  color,
+		"message":  a.Description,
+		"notify": notify,
+	})
+	if err != nil {
+		return err
+	}
+
+	resp, err := http.Post(
```

Comment:
You can add a timeout on the client, see http://golang.org/pkg/net/http/#Client

As hipchat will generally get less-critical alerts than pagerduty I think it's best to ensure it doesn't gum things up given it's all serial at the moment.